src/main/java/edu/cornell/kfs/fp/batch/xml/DisbursementVoucherPreConferenceRegistrantXml.java

```diff
@@ -0,0 +1,67 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.sys.xmladapters.KualiDecimalXmlAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "DisbursementVoucherPreConferenceRegistrant", namespace = StringUtils.EMPTY)
+public class DisbursementVoucherPreConferenceRegistrantXml {
+    
+    @XmlElement(name = "name", namespace = StringUtils.EMPTY, required = false)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String name;
+    
+    @XmlElement(name = "department_code", namespace = StringUtils.EMPTY, required = false)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String departmentCode;
+    
+    @XmlElement(name = "pre_conference_request_number", namespace = StringUtils.EMPTY, required = false)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String preConferenceRequestNuumber;
```

Comment:
Typo Nuumber => Number. Also, the Getter and setter names/variables can be updated.